zoo-css-challenge/index.html

```diff
@@ -83,49 +86,54 @@ <h2>News</h2>
 			<div class="card">
 				<time>2010-04-16</time>
 				<h4>Helping wild turtles in Seychelles</h4>
-				<a>Find out More</a>
+				<button class="card1">Find out More</button>
 			</div>
 			<div class="card">
 				<time>2010-04-12</time>
 				<h4>Learning about the rainforest</h4>
-				<a>Find out More</a>
+				<button class="card2">Find out More</button>
+			</div>
+			<div class="card">
+				<time>2021-05-31</time>
+				<h4>Which big cat are you</h4>
+				<button class="card3">Find out More</button>
 			</div>
 		</section>
 
 		<section class="education">
 			<div class="education-container">
 				<h3>Learning resources for schools</h3>
 				<p>Explore our activities, videos, and lesson plans to support a wide range of curriculum topics</p>
-				<button>Find them here</button>
+				<button class="card4">Find them here</button>
 			</div>
 		</section>
 
-		<section class="membership">
+		<section class="learning">
 			<h2>Are there any benefits or discounts for members?</h2>
 			<p>There are <strong>AMAZING</strong> benefits of becoming a member with Birmingham Zoo! Our members get special discounts in our shops, cafes, and unlimited entry</p>
```

Comment:
This is just missing a css rule to make its width 700px and centre it using the margin property